examples/scripts/ConfigureRemotingForAnsible.ps1

```diff
@@ -31,6 +31,7 @@
 # Updated by Chris Church <cchurch@ansible.com>
 # Updated by Michael Crilly <mike@autologic.cm>
 # Updated by Anton Ouzounov <Anton.Ouzounov@careerbuilder.com>
+# Updated by Nicolas Simond <contact@nicolas-simond.com>
```

Comment:
Currently the order of versions below matches the order of "Updated by".
Not sure if that is a strict requirement though, but why break tradition ;-)